portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,87 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+#subheader {
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#header {
+    text-align: center;
+    margin: auto
 }
+
+#h1 { 
+    display: block;
+    font-size: 2em;
+    margin-top: 0.67em;
+    margin-bottom: 0.67em;
+    margin-left: 0;
+    margin-right: 0;
+    font-weight: bold;
+}
+
+#h2 { 
+    display: block;
+    font-size: 1.17em;
+    margin-top: 1em;
+    margin-bottom: 1em;
+    margin-left: 0;
+    margin-right: 0;
+    font-weight: bold;
+}
+
+#h3 { 
+    display: block;
+    font-size: .83em;
+    margin-top: 1.67em;
+    margin-bottom: 1.67em;
+    margin-left: 0;
+    margin-right: 0;
+    font-weight: bold;
+}
+
+#title {
+    background-color: lightblue;
+    margin: auto;
+    text-align: center;
+}
+
+#quote-container {
+    margin-top: 20px;
+}
+
+.circular--square {
+  border-radius: 50%;
+  width: 200px;
+  height: 200px;
+}
+
+#experiences {
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
+    background-color: #cccccc;
+}
+
+#passions {
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
+    background-color: lightyellow;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+}
+
+#bottom {
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
+}
+
+/* will be using flex containers later for better layout */
+#flexContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
nit: add a trailing newline to this file (and any other place where this red indicator occurs)

Reply:
May I know what red indicator you are referring to?

Reply:
![image](https://user-images.githubusercontent.com/60520097/75284214-30fab080-57c9-11ea-82eb-86947543da30.png)

Reply:
I actually can't see in on my side.
<img width="627" alt="Screen Shot 2020-02-25 at 12 23 34 PM" src="https://user-images.githubusercontent.com/49105963/75284463-bc744180-57c9-11ea-84cf-8c6556c035af.png">

Reply:
Well that's bizarre. I'm investigating what could be going on here.

Reply:
Let's not worry about this for now. It's not that important.